### PR TITLE
[5.0] Add single minute granularity to console scheduling

### PR DESCRIPTION
**Pull request by tylercubell**
I'd venture a guess that some folks would like to run tasks a bit faster than every 5 minutes.

### CODE REVIEW

File: src/Illuminate/Console/Scheduling/Event.php [hunk 1]
```diff
@@ -460,7 +460,17 @@ public function yearly()
 	{
 		return $this->cron('0 0 1 1 * *');
 	}
-
+
```
Comment (GrahamCampbell) [inline src/Illuminate/Console/Scheduling/Event.php]: no trailing whitespace please

**Comment by jarektkaczyk** [comment]
This is redundant I suppose, unless all you want is API verbosity. Every minute is the default behaviour:
```
	protected $expression = '* * * * * *';
```

**Comment by taiyaei** [comment]
:+1: This would be nice to have.

PR should probably be squashed to one commit?

**Comment by taylorotwell** [comment]
This is already possible. You just don't call any method and that defaults to every minute.

```
$schedule->call(function() { echo 'every minute'; });
```

**Comment by tylercubell** [comment]
Wouldn't the intent be more clear if there was a `everyMinute()` method even if it's more verbose? From reading the docs it doesn't seem obvious that leaving off the method call defaults to every minute.

**Pull request closed without merging**